### PR TITLE
Add shopware.shop to public suffix list

### PR DESCRIPTION
**Pull request by shadracnicholas**
Public Suffix

# Public Suffix List (PSL) Submission

### Checklist of required steps

* [x] Description of Organization
* [x] Robust Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (`make test`)

* [x] Each domain listed in the PRIVATE section has and shall maintain at least two years remaining on registration, and we shall keep the `_psl` TXT record in place in the respective zone(s).

__Submitter affirms the following:__ 

 * [x] We are listing *any* third-party limits that we seek to work around in our rationale such as those between IOS 14.5+ and Facebook (see [Issue #1245](https://github.com/publicsuffix/list/issues/1245) as a well-documented example)

 * [x] This request was _not_ submitted with the objective of working around other third-party limits.

 * [x] The submitter acknowledges that it is their responsibility to maintain the domains within their section. This includes removing names which are no longer used, retaining the _psl DNS entry, and responding to e-mails to the supplied address. Failure to maintain entries may result in removal of individual entries or the entire section.

 * [x] The [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines) were carefully _read_ and _understood_, and this request conforms to them.
 * [x] The submission follows the [guidelines](https://github.com/publicsuffix/list/wiki/Format) on formatting and sorting.

---

For PRIVATE section requests that are submitting entries for domains that match their organization website's primary domain, please understand that this can have impacts that may not match the desired outcome and take a long time to rollback, if at all.

To ensure that requested changes are entirely intentional, make sure that you read the affectation and propagation expectations, that you understand them, and confirm this understanding. 

PR Rollbacks have lower priority, and the volunteers are unable to control when or if browsers or other parties using the PSL will refresh or update.

(Link: [about propagation/expectations](https://github.com/publicsuffix/list/wiki/Guidelines#appropriate-expectations-on-derivative-propagation-use-or-inclusion))

 * [x] *Yes, I understand*. I could break my organization's website cookies and cause other issues, and the rollback timing is acceptable. *Proceed anyways*.
---

## Description of Organization

Shopware is an open source e-commerce software. Users can self-host or decide to use SaaS or PaaS offerings. This submission is for the PaaS platform. 

Submitter: Platform Engineer for the PaaS platform.

**Organization Website:**
https://www.shopware.com/en/

## Reason for PSL Inclusion
- Cookie security: Preventing super cookies & ensuring sites on subdomains stay separate.
- URL highlighting: Shows the shop owners that these subdomains are separate from each other.

**Number of users this request is being made to serve:**
More than 1 million monthly visitors

## DNS Verification
```
dig +short TXT _psl.shopware.shop
"https://github.com/publicsuffix/list/pull/2187"
```

## Results of Syntax Checker (`make test`)

All tests passed.

### CODE REVIEW

**Comment by wdhdev** [comment]
The rationale is valid, but it seems to be copied from my last PR 😂

**Comment by shadracnicholas** [comment]
Its basically the same reason.

File: public_suffix_list.dat [hunk 1]
```diff
@@ -15144,6 +15144,7 @@ shopitsite.com
 // shopware AG : https://shopware.com
 // Submitted by Jens Küper <cloud@shopware.com>
 shopware.store
+shopware.shop
```
Comment (groundcat) [inline public_suffix_list.dat:15147]: The sorting seems to be incorrect. `shopware.shop` should be positioned above `shopware.store` ([Guideline](https://github.com/publicsuffix/list/wiki/Guidelines#sort-your-submission-correctly-important))

**Comment by groundcat** [comment]
* [ ]   Expiration (Note: Must STAY >2y at all times)
   * [ ]  `shopware.shop` expires `2026-02-28`
 * [x]   DNS _psl entries (Note: Must STAY in place)
   * [x]  `_psl.shopware.shop` [valid](https://www.nslookup.io/domains/_psl.shopware.shop/dns-records/)
 * [x]   Tests pass
 * [ ]   Sorting
 * [ ]   Reasoning/Organization description
 * [x]   Non-personal email address

---

- Please correct the sorting.  
- Please extend the registration of `shopware.shop` to be more than 2 years from now.  
- Please elaborate on the "Reason for PSL Inclusion," including details about the use case, such as where and how this domain is used. Additionally, provide any evidence on your company website that refers to the usage of this domain. 
  - Currently, a quick Google search does not show any subdomain in use (https://www.google.com/search?q=site%3Ashopware.shop), or if you have other types of evidence, we may consider that as well.

**Comment by simon-friedberger** [comment]
* [x] Expiration (Note: Must STAY >2y at all times)
  * [x] shopware.shop expires 2027-02-28
* [x] DNS _psl entries (Note: Must STAY in place)
* [x] Tests pass
* [x] Sorting
* [x] Reasoning/Organization description
* [x] Non-personal email address

**Pull request merged**